vembrane/globals.py

```diff
@@ -104,11 +104,16 @@
 }
 
 
-def without_na(values: Iterable[Any]) -> "filter":
+T = TypeVar("T")
+
+
+def without_na(values: Iterable[T]) -> Iterable[T]:
+    """Keep only values that are not `NA`."""
     return filter(lambda v: v is not NA, values)
 
 
-def replace_na(values: Iterable[Any], replacement: Any) -> Iterable[Any]:
+def replace_na(values: Iterable[T], replacement: T) -> Iterable[T]:
+    """Replace values that are `NA` with `replacement`."""
```

Comment:
Theoretically, values are `Iterable[Union[T, NoValue]]` and the replacement may even be something entirely different. That doesn't make much sense, though, so I'm fine with this as is.

Reply:
You're right. Forget about that. Shall I fix that?

Reply:
If you have the time, sure, otherwise, I think it's not too important